foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiClient.java

```diff
@@ -129,6 +144,16 @@ private <T> T execute(
     return executeRequest(request, responseType);
   }
 
+  @Nonnull
+  private <T> OpenAiChatCompletionStream<T> stream(
+      @Nonnull final String path,
+      @Nonnull final Object payload,
+      @Nonnull final Class<T> responseType) {
+    final var request = new HttpPost(path);
```

Comment:
(Comment)

Not sure, whether "stream" should automatically infer "POST" http method.

Reply:
The regular `execute` method above also automatically infers HttpPost